client/src/components/layout/Footer.js

```diff
@@ -16,11 +16,40 @@ class Footer extends Component {
         return (
             <div style={{ marginTop: "2rem" }}>
                 <Navbar dark color="dark" expand="md">
+
+                    <nav class="navbar navbar-expand-md navbar-dark bg-dark">
```

Comment:
You can use [reactstrap](https://reactstrap.github.io/components/navbar/) to render bootstrap components. It makes it a lot easier because you don't need to remember all the class names to use, you can just render React components with props.